webservices/rest.py

```diff
@@ -563,6 +563,7 @@ def api_ui():
         'swagger-ui.html',
         specs_url=url_for('docs.api_spec'),
         PRODUCTION=env.get_credential('PRODUCTION'),
+        api_key=env.get_credential('API_UMBRELLA_SIGNUP_KEY'),
```

Comment:
the `api_key` variable is very generic. can you rename this variable to be more specific?example api_signup_key